crawl/dependencymd.py

```diff
@@ -55,6 +56,14 @@ def get_ancestors(self, dependency):
                     # rule
                     ancestors.append(self._dep_key_to_dependency[dep_key])
         return ancestors
+
+    def get_classifier(self, dependency):
+        if dependency.classifier is not None:
+            return dependency.classifier
+        else:
+            if dependency.bazel_buildable:
+                return self._jar_artifact_classifier
```

Comment:
should this be checking the Env variable?

Reply:
That is handled in the config class - this value here comes from the config class.